Worktez/worktez
# Fix create new team

### Functionality:
<-- Changed the UI of create-new-team-->

### Solution:
<-- Did some changes in create-new-team component. -->

### Risk level:

### How to test:
<-- please tell us how to test functionality/bug 
steps: 1. ng serve and firebase emulators:start
          2.  go to My Organization
          3.  go to create-new-team
          4.  there you can see the changes in UI.-->
